packages/devtools/cli/src/commands/halo/join.ts

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { CliUx, Flags } from '@oclif/core';
+import chalk from 'chalk';
+
+import { Trigger, sleep } from '@dxos/async';
+import { Client, Invitation, InvitationEncoder } from '@dxos/client';
+
+import { BaseCommand } from '../../base-command';
+
+export default class Join extends BaseCommand {
```

Comment:
I think the other side of this would be a web app or another cli right? Tests for the cli invitation commands still sounds important though, similar to the tests just written for invitations panels.

Reply:
correct @wittjosiah 
though I'd rather add it as another PR, since the goal here is to land credentials work, (so I could also land a big PR on the KUBE side)